build/Dockerfile.ci

```diff
@@ -22,6 +22,14 @@ RUN tar -C /usr/local -xzf go.tar.gz
 ENV GOBIN=/usr/local/go/bin
 ENV PATH=${PATH}:${GOBIN}
 
+WORKDIR /build/
+
+# Download client binaries
```

Comment:
Please link to https://issues.redhat.com/browse/WINC-520 where we will remove this.